web-common/src/features/dashboards/leaderboard/LeaderboardListItem.svelte

```diff
@@ -42,6 +44,10 @@
     },
   } = getStateManagers();
 
+  $: checkColor = selected
+    ? colorGetter.get($page.params.name, dimensionName, label)
+    : "gray-300";
```

Comment:
The default value returned is `gray-300` by `colorGetter`. Ternary operator might be redundant.